classifiers/query_check/communication_style_classifier/code_snippet_refinery.md

```diff
@@ -0,0 +1,16 @@
+```python
+import requests
+
+ATTRIBUTE: str = "text" # only text attributes
+MODEL_NAME: str = "KernAI/multilingual-e5-communication-style"
+REQUEST_URL: str = "https://free.api.kern.ai/inference"
+
+def communication_style_classifier(record):
+    payload = {
+        "name_model": MODEL_NAME,
```

Comment:
would also be affected by `model_name` change

Reply:
Changed, thanks!